java/core/src/java/org/apache/orc/OrcFile.java

```diff
@@ -934,6 +944,18 @@ public CompressionStrategy getCompressionStrategy() {
       return compressionStrategy;
     }
 
+    public int getCompressionZstdLevel() {
```

Comment:
IMO, we'd better hide zstd-specific parameters from the public API. At least we can encapsulate them into a wrapper class.

Reply:
Add class `org.apache.orc.OrcFile.ZstdCompressOptions`